### ISSUE TITLE
Styling of state values not working

### ISSUE THREAD

**Issue by braindead1** [Zefau/ioBroker.jarvis#13]
In v0.0.4 the styling of states is not working as expected. 

I'm using the following JSON to change the color of the icon and the state value depending on the value. The styling of the icon works for both values, but the the styling of the state value only works for "true".

``` 
{
  "anwesenheit_markus": {
    "name": "Markus",
    "function": "default",
    "states": {
      "presence": {
        "state": "0_userdata.0.anwesenheit.markus"
      }
    },
    "jarvis": {
      "group": "Anwesenheit",
      "state": {
        "true": "anwesend",
        "false": "abwesend"
      },
      "styles": {
        "icon": {
          "true": {
            "color": "purple"
          },
          "false": {
            "color": "red"
          }
        },
        "state": {
          "true": {
            "color": "purple"
          },
          "false": {
            "color": "red"
          }
        }
      },
      "icon": "account"
    }
  }
}
``` 

![styling](https://user-images.githubusercontent.com/2136249/76329682-2e31ac80-62ed-11ea-82c1-33b7bfaf3d82.png)

**Comment by Zefau** [comment]
Should be fixed with v0.0.5. Could you please test and verify? Thanks

**Comment by braindead1** [comment]
The bug is fixed in v0.0.5.
